sbncode/Cluster3D/SnippetHit3DBuilderSBN_tool.cc

```diff
@@ -707,7 +707,7 @@ size_t SnippetHit3DBuilderSBN::BuildHitPairMap(PlaneToSnippetHitMap& planeToSnip
     // Where are we?
     mf::LogDebug("SnippetHit3D") << "Total number hits: " << totalNumHits << std::endl;
     mf::LogDebug("SnippetHit3D") << "Created a total of " << hitPairList.size() << " hit pairs, counted: " << hitPairCntr << std::endl;
-    mf::LogDebug("SnippetHit3D") << "-- Triplets: " << nTriplets << ", dead channel pairs: " << nDeadChanHits << std::endl;
+    mf::LogDebug("SnippetHit3D") << "-- Triplets: " << nTriplets << std::endl;
```

Comment:
`c14` will complain about this change now. Either leave this message like it was, or also remove line 671:

`size_t nDeadChanHits(0);`

Reply:
ah crap I thought I removed that variable, done!